content/1.2-rc/docs/installation/requirements.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+## Operators
+The orchestrator runtime/deployment is made of two main parts: `sonataflow operator` and `RHDH operator`  
+SonataFlow operator resource requirements are described [SonataFlow Installation Requirements](https://sonataflow.org/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html#_prepare_a_minikube_instance)  
+The operator deploys a Data Index service and a Jobs service.
+These are the recommended minimum resource requirements for their pods:
+```
+Data Index pod:
```

Comment:
for styling, pull the "Data Index pod:" outside of the section and use yaml:

**Data Index pod:**
```yaml
resources:
  limits:
    cpu: 500m
     memory: 1Gi
  requests:
    cpu: 250m
    memory
```

Reply:
fixed